framework/decode/vulkan_json_util.h

```diff
@@ -50,15 +50,15 @@ struct JsonOptions
 };
 
 template<typename T>
-constexpr std::string to_hex_variable_width(T value)
+std::string to_hex_variable_width(T value)
```

Comment:
I remember being surprised that these could be `constexpr` when I wrote them. I guess they couldn't after all.